client/src/components/ConfirmationDialog.js

```diff
@@ -53,7 +53,9 @@ const ConfirmationDialog = ({ isOpen, onClose, confirm, itemType, item, action,
                                 }
                                 toast({
                                     title: `${action} ${itemType}  `,
-                                    description: `${action} ${itemType} successful`,
+                                    description: `${itemType} '${additionalDetail === "" ? "" : additionalDetail + " "}${
```

Comment:
i think you can just do 

`${additionalDetail &&  additionalDetail + " "}`

And then you don't have to pass in an empty string.

Reply:
so what i thought while doing this is that if there is additional details being passed in then it formats it i.e. softeng254 is a additional detail so that the toast shows 'SOFTENG762 Agile has been deleted' however if no additional detail is passed in then it would be like '(space here) Agile' has been deleted.